opentelemetry-sdk/src/logs/batch_log_processor.rs

```diff
@@ -39,7 +39,7 @@ use std::{
 /// Delay interval between two consecutive exports.
 pub(crate) const OTEL_BLRP_SCHEDULE_DELAY: &str = "OTEL_BLRP_SCHEDULE_DELAY";
 /// Default delay interval between two consecutive exports.
-pub(crate) const OTEL_BLRP_SCHEDULE_DELAY_DEFAULT: u64 = 1_000;
+pub(crate) const OTEL_BLRP_SCHEDULE_DELAY_DEFAULT: Duration = Duration::from_millis(1_000);
```

Comment:
should we also change it for `OTEL_BLRP_EXPORT_TIMEOUT_DEFAULT`?

Reply:
oh yes sure!